src/test/java/com/rarchives/ripme/tst/ripper/rippers/InstagramRipperTest.java

```diff
@@ -23,15 +27,15 @@ public void testInstagramGID() throws IOException {
         }
     }
 
-    /*
     public void testInstagramAlbums() throws IOException {
         List<URL> contentURLs = new ArrayList<>();
-        contentURLs.add(new URL("http://instagram.com/anacheri"));
```

Comment:
Why remove a test that actually rips a whole profile page?

Reply:
> Why remove a test that actually rips a whole profile page?

Because it was randomly failing, I tried replacing it with other profile links but they also failed at random

Reply:
Fair enough